.github/actions/openvino_provider/action.yml

```diff
@@ -0,0 +1,157 @@
+name: 'OpenVINO provider'
+description: 'Provides OpenVINO build artifacts from a requested source'
+inputs:
+  revision:
+    description: 'Version of OpenVINO to provide - latest_available | HEAD | specific commit hash'
+    required: true
+  branch_name:
+    description: 'Branch of OpenVINO to take the revision from if no specific hash was provided. 
+                  Taken from github context by default'
+    required: false
+  ov_artifact_name:
+    description: "Name under which to upload provided OpenVINO build artifacts, set automatically by default"
+    required: false
+  commit_packages_to_provide:
+    description: "Comma-separated list of OpenVINO packages from post-commit to provide ('all' to get everything)"
+    required: false
+  commit_build_key:
+    description: 'public_linux_ubuntu_20_04_release | public_linux_ubuntu_22_04_release | 
+                  public_linux_ubuntu_24_04_release | public_windows_vs2019_release'
+    required: false
+  commit_share_path:
+    description: 'Path to share holding OpenVINO post-commit artifacts'
+    required: false
+    default: '/mount/build-artifacts'
```

Comment:
Will it throw an error if the default share is not mounted?

Reply:
It will throw an error when it tries to access the folder on this share